src/main/java/org/jabref/gui/sidepane/SidePaneComponent.java

```diff
@@ -71,4 +72,13 @@ private Node createHeaderView() {
     protected void addExtraButtonToHeader(Button button, int position) {
         this.buttonContainer.getChildren().add(position, button);
     }
+
+    public void requestFocus() {
+        for (Node child : getChildren()) {
+            if (child instanceof GroupTreeView) {
+                ((GroupTreeView) child).requestFocusGroupTree();
+                break;
+            }
+        }
```

Comment:
Casting can be omitted by the new pattern matching feature of java
```suggestion
        for (Node child : getChildren()) {
            if (child instanceof GroupTreeView groupTreeView) {
                groupTreeView.requestFocusGroupTree();
                break;
            }
        }
```

Reply:
casting has been replaced by pattern matching